README.md

```diff
@@ -33,6 +33,25 @@ Developers and anyone else with a PyTorch or TensorFlow neural network model at
 ## Attacks
 
 ### Property Inference Attack
+![plot](docs/Property_Interference_Attacks.png)
+
+The Property Inference Attack aims to detect patterns in the parameters of the target model including properties 
+which the model producer has not intended to reveal. Therefore, the adversary requires a 
+meta-classifier which predicts one of these properties he is interested in. 
+
+The first step of the attack is to generate a set containing k datasets for k shadow classifiers which then, are trained 
+to build the training set for the meta classifier. These datasets could be created by sampling from a larger dataset or
+by integrating more data. Furthermore, it is crucial that one half of the shadow classifiers 
```

Comment:
It does not become clear what you mean by "integrating more data" - where should it come from? And where should it be integrated to?

Reply:
This part has been updated in commit 9ea9416.